src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px ;
+  --text-color: #060b35;
+  --active-color: #00acdc;
+  font-family: "Roboto", sans-serif;
```

Comment:
1) Ok, you have variables for main colors but you don't use them in other blocks🤷‍♂️
Let's move them to the variables file and use SASS variables instead of CSS.
2) The main font family should be applied to the html or body, also don't forget about fallback font.
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/e8aa1121-ce8d-45b8-955f-40c93678ccc6)